forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+  to the appendix that uses the cumulative mean as suggested (the sum is
+  difficult to see due to the scaling). With some exceptions at the beginning of
+  the evaluation period, the relative ordering remains fairly consistent. The AR
+  model is easily the worst.
+
+  TODO I have added the code and a caption for the figure to the Supplement, but
+  no corresponding 
+  text. It seems a fairly minor addition, not necessarily deserving its own
+  section. 
+\end{resp}
+
+2. Are the results in this paper statistically significant? I see no discussion
+of this question, which seems unbelievable for me to be saying, given the
+authorship. And yet here I am.
+
+\begin{resp}
+  This is an important point that we should have addressed more fully.
+  Conventional methods for assessing statistical significance that we explored
+  were overconfident, to the point of being unbelievable. To a large degree,
+  this is a ``huge n, tiny p'' problem so even bootstrap analyses result in
+  extremely narrow intervals. Essentially, ``everything is significant''.
+
+  We also have strong reasons for not using model-based approaches (one being the
+  nonstationarity the Reviewer mentions above, but also the belief that these
+  models are wrong). We felt that the related investigation in 2B was more
+  pertinent.
+
+  To respond more directly:
+
+  $\bullet$ We have added a section to the Supplement (and point
+  to it around 
+  line 85 in the main text), that performs a Sign Test for differences in
+  accuracy. P-values there tend to cluster near zero suggesting again that these
+  improvements are statistically significant.
+
+  $\bullet$ We also use a Diebold-Mariano test
+  for comparing forecast accuracy. The results for this test are more mixed,
+  likely due to nonstationarity the Reviewer mentions above (the indicators are
+  more helpful in some periods than others, see also the upswing/downswing
+  discussion). 
+\end{resp}
+
+3. Are the results in this paper practically significant? I see no discussion of
+this question, which seems unbelievable for me to be saying, given that the
+authors refuse to consider statistical significance. And yet here I am.
+
+\begin{resp}
+  See also Reviewer 1 point 3. And the general comment below (Point 8).
+  Practically, on average, if our goal is to predict 11 days ahead, when the AR
+  model is at its best, CHNG-CLI
+  doesn’t help (at least with these simple models). However, CHNG-COVID,
+  DV-CLI, and Google-AA ``buy'' 4 extra days for the 
+  same accuracy. CTIS-CLIIC ``buys'' about 5 extra days. What we mean here is
+  that if we have a certain tolerance for inaccuracy, adding these signals increases the
+  horizon at which we can predict within our tolerance. As for the
+  practical significance of 4-5 extra days, this would of course depend on the use case.  However, cases grow
+  exponentially, so making public policy changes 15 or 16 days ahead rather than
```

Comment:
This response is consistent with how we've explained the "practical gains" of our indicator-assisted models to ourselves in the past, but the specific response that one would have (e.g.) four extra days of lead time in preparing for an upswing is implicitly blunted by the fact that the indicator-assisted models don't have a clear advantage in predicting upswings (vs downswings).  

Just pointing out what would be my immediate reaction if I were this reviewer.  Maybe we double down on the view that this paper isn't meant to debut the best forecasting models but rather demonstrate a framework within which to evaluate competing forecasting models?

Reply:
But looking at the new Figure S21 (cumulative mean), doesn't it look like the indicator-assisted models have an advantage over the AR model for predicting upswings?  All models find upswings difficult to predict, but indicator-assisted ones do a better job?

Reply:
If I understand this plot correctly (and focusing on the upswing from Oct 1 - Dec 31st, then the general downward movement of the cumulative mean WIS suggests that the AR[+indicator] models do better than the baseline during upswings; but, the narrowing gap between the black curve (AR) other colors (indicator-assisted) suggests that the indicators are "barely helping" during the upswings, right?  [To get a large gap {<=> indicators are really helping} in the cumulative mean, the indicators need to be putting small WIS terms into the average relative to the AR model.  So the narrowing of the gap suggests that during that upswing the indicator-assisted models are adding a bunch of terms relatively the same size as the plain AR model to the cumulative mean?]

In any case, my general comment is still a reviewer may think it's misleading to use the overall (averaged over all dates) result on "days bought" by an indicator to claim that one would be able to "buy" that many days in an upswing... I might just be overthinking it

![Screen Shot 2021-09-28 at 20 40 32](https://user-images.githubusercontent.com/8422804/135183976-eb330194-2235-44fb-a607-4e3a13f0025d.png)

Reply:
Add GeoMean figure to Supplement as well.

Reply:
Add week ticks (even maybe hashes on x-axis).